pkg/webrtctransport.go

```diff
@@ -159,17 +164,35 @@ func (p *WebRTCTransport) AddICECandidate(candidate webrtc.ICECandidateInit) err
 
 // OnICECandidate handler
 func (p *WebRTCTransport) OnICECandidate(f func(c *webrtc.ICECandidate)) {
+	p.mu.Lock()
```

Comment:
Do we need to guard here? Since it is just calling to pc

Reply:
Removed.